### PR TITLE
Feature: add groups to feature in admin

### LINKED ISSUE #13: Add Groups to Feature in the admin
As an admin I should be able to add groups to the features

### PR DESCRIPTION
**Pull request by martinjaimem**
#### Description
As an admin I should be able to add/enable groups to the features

Screen Shot 2020-07-31 at 11 38 30
Screen Shot 2020-07-31 at 11 38 36

### CODE REVIEW

File: django_feature_flip/admin.py [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+
+def groups_list(feature):
+    return [group.name for group in feature.groups.all()]
 
 
 class FeatureAdmin(admin.ModelAdmin):
     list_display = (
         'name',
         'totally_enabled',
+        groups_list
```
Comment (guilleijo) [inline django_feature_flip/admin.py:13]: Trailing comma

File: django_feature_flip/admin.py [hunk 2]
```diff
@@ -2,10 +2,15 @@
...
 
+def groups_list(feature):
+    return [group.name for group in feature.groups.all()]
+
+
 class FeatureAdmin(admin.ModelAdmin):
     list_display = (
         'name',
         'totally_enabled',
+        groups_list,
```
Comment (kaozdl) [inline django_feature_flip/admin.py:13]: this groups list is the function itself, not the result.
Is that what you actually want?

File: django_feature_flip/models.py [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+    name = models.CharField(max_length=40, unique=True, null=False)
+
+    def __str__(self):
+        return self.name
+
+
 class Feature(models.Model):
-    name = models.CharField(max_length=40, unique=True)
-    totally_enabled = models.BooleanField(default=False)
+    name = models.CharField(max_length=40, unique=True, null=False)
```
Comment (kaozdl) [inline django_feature_flip/models.py:12]:
```suggestion
    name = models.CharField(max_length=40, unique=True, db_index=True, null=False)
```

File: django_feature_flip/admin.py (hunk 2 above)
Comment (martinjaimem) [reply django_feature_flip/admin.py:13]: Yes, I mean, it does the trick. Would you like to have it written differently?

File: django_feature_flip/helpers.py [hunk 4]
```diff
@@ -43,7 +43,7 @@ def activate_group(self, feature_name, group_name):
         get_group_or_raise_error(group_name).features.add(feature)
 
     def feature_enabled_for_a_group_of_actor(self, feature, actor):
-        groups_names = [group.name for group in feature.group_set.all()]
+        groups_names = [group.name for group in feature.groups.all()]
```
Comment (kaozdl) [inline django_feature_flip/helpers.py:46]:
```suggestion
        groups_names = feature.groups.all().values_list('name', flat=True)
```

File: django_feature_flip/models.py [hunk 5]
```diff
@@ -1,17 +1,17 @@
 from django.db import models
 
 
-class Feature(models.Model):
-    name = models.CharField(max_length=40, unique=True)
-    totally_enabled = models.BooleanField(default=False)
+class Group(models.Model):
+    name = models.CharField(max_length=40, unique=True, null=False)
```
Comment (kaozdl) [inline django_feature_flip/models.py:5]:
```suggestion
    name = models.CharField(max_length=40, unique=True, db_index=True, null=False)
```

**Pull request merged**